src/postamble.js

```diff
@@ -8,11 +8,12 @@
 
 {{{ exportRuntime() }}}
 
-#if MEM_INIT_IN_WASM == 0
+#if !MEM_INIT_IN_WASM
+function runMemoryInitializer() {
```

Comment:
I'm not sure how to read this code. This new function starts here, but the PR doesn't seem to end it? Maybe I'm missing something.

Reply:
It looks funny because I create new function, but removed the internal nesting within the function, replacing it with an early return.

So the overall level of nesting remained constant even though a function was added.